server/src/assets/game/flags-swtbahn-full.js

```diff
@@ -0,0 +1,242 @@
+const signalToFlagFull = {
+    "signal4": "flagThemeBlack flagOutline flagOne",
+    "signal18": "flagThemeBlack flagOutline flagTwo",
```

Comment:
Examples of the mapping strings that should be generated instead.
Each string defines the colour theme, the type of flag design, and the flag symbol.